### ISSUE TITLE
I'm begin study moqui. In quick tutorial not allow edit form

### ISSUE THREAD

**Issue by imxingquan** [moqui/moqui-framework#390]
In https://www.moqui.org/docs/framework/Quick+Tutorial#AutomaticFindForm

the edit dialog show tutorial entity fields, but it's not allowd edit! Why?
![image](https://user-images.githubusercontent.com/14246824/68170083-5e68e780-ffa9-11e9-801b-9d6dbd67eced.png)

**Comment by zhangwei1979** [comment]
Check if you missed transition below

<transition name="createTutorial">
    <service-call name="create#tutorial.Tutorial"/>
    <default-response url="."/>
</transition>

**Comment by imxingquan** [comment]
I have this code
![image](https://user-images.githubusercontent.com/14246824/68567081-14916d00-0493-11ea-8443-7a6237394af9.png)

**Comment by zhangwei1979** [comment]
Try require-authentication="false"

**Comment by imxingquan** [comment]
use require-authentication="false"  when I visit url 'http://localhost:8080/apps/tutorial/FindTutorial' show error tips:
![image](https://user-images.githubusercontent.com/14246824/68634472-8ff52c00-0530-11ea-9e7b-c84ee6f0c3ad.png)
I click Test Login button after visit up url. or visit this url directly display this error message.

**Comment by growerp** [comment]
Good morning,

we have some students here doing the same as you do please have a look at:
https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/moqui/BrHL7d1-1Vo/363JUyBuAgAJ
which proposes some changes to the Quick tutorial.

regards,

Hans Bakker
antwebsystems.com

**Comment by jonesde** [comment]
The Quick Tutorial hadn't seen a refresh in many years... pretty much the same content that was in Making Apps with Moqui from 2014. Kind of a lot has changed since then, including recommended practices because better ways of doing various things are now available.

Anyway, I did a big refresh on it today that includes a section (under "Make a Real Application") about ArtifactAuthz setup. It could have been updated with other security workarounds, but this is a useful topic to cover in the tutorial. There are a half dozen or so other changes for recommended ways of doing things and filling on gaps on creating new components and apps and such.

https://www.moqui.org/m/docs/framework/Quick+Tutorial
